### PR TITLE
Update Opera support for angle CSS type

### PR DESCRIPTION
**Pull request by rudimadima**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by ddbeck** [comment]
Hi @rudimadima, I see this is your first PR here—welcome to BCD!

Can you provide any more background for these changes? Did you test these yourself or have documentation you could share? We want to make sure that the data that gets into BCD is substantiated.

Thank you!

**Comment by rudimadima** [comment]
Hello.
I tested CSS-angles in old Opera Presto 12. It works correctly.

**Comment by vinyldarkscratch** [comment]
Hey @rudimadima, thanks for the response!  Could you elaborate on the details, such as:

- How did you test for support?
- Did you test in older versions as well?

**Comment by rudimadima** [comment]
Hello.
I tested it only in Opera **12.17**

**Comment by vinyldarkscratch** [comment]
Okay, if you had only tested it in 12.17, let's set a ranged version, "≤12.1", for both Opera and Opera Android.  Additionally, __how__ did you test for support?

**Comment by rudimadima** [comment]
By using rotate function
like:
`transform: rotate(-20deg);
transform: rotate(110grad);
transform: rotate(1.2rad);
transform: rotate(0.14turn);
`

File: css/types/angle.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "12"
```
Comment (ddbeck) [inline css/types/angle.json:28]: This ought to be changed to `"≤12.1"` and applied to all the versions in this diff:

```suggestion
              "version_added": "≤12.1"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for the update, @rudimadima. A small adjustment to the version data is needed; a comment in line demonstrates. After these changes are made, we can merge this. Thank you again!

**Comment by ddbeck** [comment]
Hi @rudimadima, do you plan to return to this PR?

**Comment by ddbeck** [comment]
Since this PR has been inactive for a long time, I'm going to close it. If you'd like to eventually come back to this, feel free to open a new PR. Thank you!

**Pull request closed without merging**